p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/Steps/RestoreCustom/RestoreCustomDelegateCoordinator.swift

```diff
@@ -52,7 +55,6 @@ final class RestoreCustomDelegatedCoordinator: DelegatedCoordinator<RestoreCusto
 
     private func openHelp() {
         openInfo()
```

Comment:
helpLauncher.launch()

Reply:
openInfo - openHelp

Reply:
@lisemyon what do you mean?

Reply:
we made a call a discussed this method. it should open help but now it opens terms. we looked through all screens and there are several places which requires intercom also